src/main/java/com/breskul/bibernate/util/EntityUtil.java

```diff
@@ -251,27 +330,121 @@ public static Class<?> getEntityCollectionElementType(Field field) {
   }
 
 
-  //todo check if we should use getClassColumnFields instead this one
-  public static <T> List<Field> getEntityColumns(Class<? extends T> entityClass) {
-    return Arrays.stream(entityClass.getDeclaredFields())
-        .filter(field -> field.isAnnotationPresent(Column.class))
-        .collect(Collectors.toList());
+  /**
+   * Retrieves an instance of the Collection class based on the given Field representing a
+   * collection.
+   *
+   * @param collectionField - The Field representing the collection
+   * @return An instance of the Collection class
+   * @throws IllegalArgumentException if the collection type is unsupported
+   */
+  public static Collection<Object> getCollectionInstance(Field collectionField) {
+    var collectionClass = collectionField.getType();
+
+    if (collectionClass.isAssignableFrom(List.class)) {
+      return new ArrayList<>();
+    }
+
+    if (collectionClass.isAssignableFrom(Set.class)) {
+      return new HashSet<>();
+    }
+
+    throw new IllegalArgumentException(
+        "Unsupported collection: " + collectionClass); // change exception and more clear msg?
+  }
```

Comment:
This method is present in `AssociationUtil` class, please merge `develop` branch into your branch first.

Reply:
It was added by mistake during resolution of merge conflicts

Reply:
removed